### PR TITLE
Update to latest ingress-nginx

### PR DESCRIPTION
**Pull request by timm088**
#### What this PR does / why we need it:

Bumps the version to the latest available

#### Which issue this PR fixes
N/A - Incremental version increase

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X ] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [X ] Chart Version bumped
- [X ] Variables are documented in the README.md

### CODE REVIEW

File: stable/nginx-ingress/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: nginx-ingress
-version: 1.1.5
-appVersion: 0.21.0
+version: 1.2.0
+appVersion: 0.22.0
```
Comment (davidkarlsen) [inline stable/nginx-ingress/Chart.yaml:3]: please update the readme too for consistency.

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/nginx-ingress/Chart.yaml (hunk 1 above)
Comment (timm088) [reply stable/nginx-ingress/Chart.yaml:3]: Updated readme default

**Comment by timm088** [comment]
Ready for review again thanks @davidkarlsen

**Review by mattfarina — APPROVED** [review]

**Comment by avik-so** [comment]
I want to bring it to the relevant people's attention that this version update had a breaking change.
Is there a way to deploy version 0.21.0 or any specific version?

**Comment by timm088** [comment]
@avik-so you will need to target a specific chart version, that is why charts are versioned.
https://github.com/helm/helm/blob/master/docs/helm/helm_install.md

Check the --version flag.

**Comment by TBBle** [comment]
Shouldn't a breaking change in the App have bumped the chart to 2.x? Or is that not what the Helm chart's SemVer is supposed to advertise?

**Pull request merged**